ksql-execution/src/main/java/io/confluent/ksql/execution/function/udtf/KudtfFlatMapper.java

```diff
@@ -28,20 +29,32 @@
 @Immutable
 public class KudtfFlatMapper implements ValueMapper<GenericRow, Iterable<GenericRow>> {
```

Comment:
Do we have a unit test for this class? I'd expect some more detailed unit tests around this behavior than just QTT

Reply:
Added